FreeRTOS/FreeRTOS-Plus-TCP
# Fix TCP IP failing unit test cases

Description
-----------
This PR fixes the TCP IP unit test cases that were failing after the IPv6 related source code changes.

Test Steps
-----------
```
cmake -S test/unit-test -B test/unit-test/build/ 
make -C test/unit-test/build/ all 
cd test/unit-test/build/
ctest -E system --output-on-failure --verbose
```
Related Issue
-----------
Failing TCP IP test cases.

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.
